storage/local/storage.go

```diff
@@ -212,10 +212,18 @@ func (s *memorySeriesStorage) Start() error {
 		return err
 	}
 	s.persistence = p
+	go s.persistence.run()
+
+	pclose := func() {
```

Comment:
That's what I had initially. Then I found the risk that someone comes and overwrites `err` with a `:=` greater than than the one to forget `pclose()`.